ecoroofs/static/pages/page.html

```diff
@@ -1,18 +1,16 @@
 <md-content class="standard-padding" ng-cloak>
 
-    <h2 ng-click="$ctrl.openSidenav()" class="md-title">{{ $ctrl.page.name }}</h2>
```

Comment:
@brennenf 
This change makes the page's title (name) no longer visible. If we don't need both name and description, I'd suggest removing description as the name seems most meaningful.